client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -41,17 +41,32 @@ export const JetpackSitesDataViews = ( {
 	const sitesPerPage = dataViewsState.perPage > 0 ? dataViewsState.perPage : 20;
 	const totalPages = Math.ceil( totalSites / sitesPerPage );
 
-	const sites = useFormattedSites( data?.sites ?? [] );
+	const sites = useFormattedSites( data?.sites ?? [] ).reduce< SiteData[] >( ( acc, item ) => {
+		acc.push( item );
+		// If this site has an error we should add a row to display an error message below this site's row
+		if ( item.site.error ) {
```

Comment:
Not sure I understand we need it here. Can we just use `item.site.error` in the code below? It also seems like we will have this `item` pushed twice into `acc`

Reply:
You are right, when we have a site error the site gets pushed twice into `acc`. We do this to duplicate the row, changing the second row's type to `error`, which will signal the render to render the duplicated row as an error message.

Reply:
Tried to improve this comment here: e957b08

Is it clearer now?